### PR TITLE
chore: refactor for removing pdb symbols

### PR DESCRIPTION
**Pull request by peppelinux**
This PR represents an easy refactor of small parts of the source code where the PDB symbols was used for variable assignments. It simply removed the variable names `a`, `c` that are used in PDB to inspect the args and continue the execution.

Closes https://github.com/latchset/jwcrypto/issues/327#issuecomment-1686352603

### CODE REVIEW

File: jwcrypto/jwk.py [hunk 1]
```diff
@@ -1028,26 +1028,26 @@ def export_to_pem(self, private_key=False, password=False):
...
         :rtype: `bytes`
         """
-        e = serialization.Encoding.PEM
+        _e = serialization.Encoding.PEM
         if private_key:
             if not self.has_private:
                 raise InvalidJWKType("No private key available")
             f = serialization.PrivateFormat.PKCS8
             if password is None:
-                a = serialization.NoEncryption()
```
Comment (simo5) [inline jwcrypto/jwk.py:1037]: and this a can be change to alg for algorithm

**Comment by simo5** [comment]
@peppelinux any chance you can fix the nits and push anew? I'd like to merge this.

**Comment by peppelinux** [comment]
Ciao @simo5 
I just have pushed my commit with your revision, thank you for the reminder, I just needed that!

**Comment by simo5** [comment]
@peppelinux great, would you mind squashing the commits together, I will then merge.

**Review by simo5 — APPROVED** [review]

**Comment by simo5** [comment]
Uhmm sounds like the linter is not happy, if "aa" does not work, feel free to change to "aad"

**Comment by peppelinux** [comment]
Done @simo5 !

File: jwcrypto/jwa.py [hunk 2]
```diff
@@ -887,18 +887,18 @@ def __init__(self, hashfn):
...
+        h.update(ai)
         h.update(iv)
         h.update(e)
         h.update(al)
         m = h.finalize()
         return m[:_inbytes(self.keysize)]
 
     # RFC 7518 - 5.2.2
-    def encrypt(self, k, a, m):
+    def encrypt(self, k, ai, m):
```
Comment (simo5) [inline jwcrypto/jwa.py:901]: this "ai" and others need to be renamed to "aad" too, see lint failures indicating it.

**Review by simo5 — CHANGES_REQUESTED** [review]
Unfortunately the linter is still not 100% happy, 4 places need to change ai->aad

File: jwcrypto/jwa.py (hunk 2 above)
Comment (simo5) [reply jwcrypto/jwa.py:901]: no hurry

Comment (peppelinux) [reply jwcrypto/jwa.py:901]: Done, tested locally with pytest and pylint. it looks good to me

**Review by simo5 — APPROVED** [review]
LGTM, thanks for the contribution

**Pull request merged**